### PR TITLE
update: move Hands-on start page to the Kotlin docs

### PR DESCRIPTION
**Pull request by koshachy**
https://youtrack.jetbrains.com/issue/KT-58683/Move-Kotlin-Hands-On-overview-page-to-the-Docs

### CODE REVIEW

File: docs/topics/kotlin-hands-on.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+[//]: # (title: Kotlin hands-on)
+
+A series of hands-on labs where you can create applications with Kotlin using a variety of different technologies and targeting multiple platforms.
```
Comment (danil-pavlov) [inline docs/topics/kotlin-hands-on.md:3]:
```suggestion
A series of hands-on tutorials where you can create applications with Kotlin using a variety of different technologies and targeting multiple platforms.
```

**Review by danil-pavlov — APPROVED** [review]

File: docs/topics/kotlin-hands-on.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+[//]: # (title: Kotlin hands-on)
```
[the remark this answers is not in the document]
Comment (koshachy) [reply docs/topics/kotlin-hands-on.md:1]: Discussed in Slack. Refused.

**Comment by koshachy** [comment]
@nikpachoo Please ping me when you will be ready to merge redirects

File: docs/topics/kotlin-hands-on.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+A series of hands-on tutorials where you can create applications with Kotlin using a variety of different technologies and targeting multiple platforms.
+The exercises are divided into a series of steps, walking you through each section.
+
+## Building Reactive Spring Boot applications with Kotlin coroutines and RSocket
+
+The tutorial shows you how to build a simple chat application using Spring Boot and Kotlin.
+You will learn about the benefits of using Kotlin for server-side development from a syntax perspective.
```
Comment (sarahhaggarty) [inline docs/topics/kotlin-hands-on.md:9]:
```suggestion
Build a simple chat application using Spring Boot and Kotlin, and learn about the benefits of using Kotlin for server-side development from a syntax perspective.
```

File: docs/topics/kotlin-hands-on.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+## Building web applications with React and Kotlin/JS
+
+Create a React Application using Kotlin/JS, and see how you can leverage Kotlin's type system, library ecosystem, and interoperability features.
+
+[**Start**](js-react.md)
+
+## Building web applications with Spring Boot and Kotlin
+
+This tutorial shows you how to build efficiently a sample blog application by combining the power of Spring Boot and Kotlin.
```
Comment (sarahhaggarty) [inline docs/topics/kotlin-hands-on.md:21]:
```suggestion
Build a sample blog application by combining the power of Spring Boot and Kotlin.
```

File: docs/topics/kotlin-hands-on.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Creating HTTP APIs with Ktor
+
+Create a backend API for your application that responds to HTTP requests.
+
+[**Start**](https://ktor.io/docs/creating-http-apis.html)
+
+## Creating a WebSocket chat with Ktor
+
+Learn how to create a simple Chat application using Ktor including both a JVM server and a JVM client.
```
Comment (sarahhaggarty) [inline docs/topics/kotlin-hands-on.md:33]:
```suggestion
Create a simple Chat application using Ktor including both a JVM server and a JVM client.
```

**Review by sarahhaggarty — APPROVED** [review]
Nice! i've made some suggestions to make the introductions look more uniform but it's up to you.

**Pull request merged**